{
    "client_datetime": "2024-08-12 17:13:46",
    "server_datetime": "2024-08-12 20:13:46",
    "data": [
        [
            "2024-08-12 14:04:52",
            "A123EE0",
            "erro_outros",
            "",
            "",
            "send_request_client_socket",
            "<class _ConnectionRefusedError_>",
            "[WinError 10061] No connection could be made because the target machine actively refused it"
        ],
        [
            "2024-08-12 14:04:54",
            "A123EE0",
            "erro_outros",
            "",
            "",
            "send_request",
            "",
            "maximum number of attempts to connect to server exceeded"
        ],
        [
            "2024-08-12 14:04:54",
            "A123EE0",
            "erro_outros",
            "",
            "",
            "request_datetime_server",
            "<class _UnboundLocalError_>",
            "cannot access local variable _response_ where it is not associated with a value"
        ],
        [
            "2024-08-12 14:21:11",
            "A123EE0",
            "erro_outros",
            "",
            "",
            "send_request",
            "<class _ConnectionResetError_>",
            "[WinError 10054] An existing connection was forcibly closed by the remote host"
        ],
        [
            "2024-08-12 14:21:13",
            "A123EE0",
            "erro_outros",
            "",
            "",
            "send_request_client_socket",
            "<class _OSError_>",
            "[WinError 10038] An operation was attempted on something that is not a socket"
        ]
    ]
}